index.html

```diff
@@ -12,24 +12,30 @@
 </head>
 <body>
   <div class="status-bar">
-    <div class="hour">9:41</div>
-    <div class="wifi"></div>
-    <div class="signal"></div>
-    <div class="battery"></div>
+    <div class="status-hour">9:41</div>
+    <div class="status-bar-space"></div>
+    <div class="status-images">
+      <img src="./Images/status.png">
+    </div>    
   </div>
-  <nav class="navbar">
-     
+  <nav class="navbar">     
     <div class="nav-name">Jhon Doe</div>
-    <div class="nav-menu"></div>      
```

Comment:
- [ ] Kindly note that your logo should be made a link as it is usually used to navigate back to the home page when someone has navigated to other pages. Changing it like below will fix it.
```
        <h1 class="logo-txt"><a href="#">My Logo</a></h1>
```

- [ ] Also the logo should not be inside a ```div``` kindly put it in a ```hx``` tag.